.vscode/launch.json

```diff
@@ -67,16 +55,10 @@
       "mode": "auto",
       "preLaunchTask": "Build Radius (all)",
       "program": "${workspaceFolder}/cmd/controller/main.go",
-      "cwd": "${workspaceFolder}",
       "args": [
-        "--config-file",
-        "${workspaceFolder}/cmd/controller/controller-self-hosted.yaml",
         "--cert-dir",
         ""
       ],
-      "env": {
-        "RADIUS_ENV": "self-hosted"
-      }
```

Comment:
This was all simplified and made consistent. There's less configuration here because we're relying on the defaults.